docs/index.md

```diff
@@ -1608,12 +1608,15 @@ by the cloud provider.
 | CentOS                     | 7+               | experimental            |
 | Fedora                     | 21+              | experimental            |
 
-If you want to use a different base operating system on a remote provider,
-you will need to select the image accordingly for that provider.  For
-example, on DigitalOcean you would use the `--digitalocean-image` flag.
-For Amazon AWS, you would use the `--amazonec2-ami` flag.
+To use a different base operating system on a remote provider, supply the
+provider's image flag and an image available from the provider.  Use the
```

Comment:
@thaJeztah Found, the two space problem exists throughout the file. If I'm going to kill those nits, I'll pick off the head casing as well. Ta for the comments guys. @nathanleclaire Yes, you are right. Good catch. Reworded.